.travis-build.sh

```diff
@@ -88,14 +72,15 @@ if [[ "${GROUP}" == "downstream" || "${GROUP}" == "all" ]]; then
   else
     CFISLUGOWNER=${SLUGOWNER}
   fi
+  CFISLUGOWNER=smillst
```

Comment:
This should be changed before being merged.

Reply:
Yes, it's related to the change below.